scripts/accept_licenses.sh

```diff
@@ -0,0 +1,7 @@
+mkdir "$ANDROID_HOME/licenses" || true \
```

Comment:
Could you add an explanation for why we're doing this and how we get these hashes (and/or a link to an online resource)

Reply:
Added some explanation in fe5b3d71a10967c157d09e9e39b63e6f876f2d29 Let me know if it needs more clarification!